fastlane/lib/fastlane/actions/testfairy.rb

```diff
@@ -76,7 +81,36 @@ def self.available_options
                                        short_option: '-g',
                                        env_name: "FL_TESTFAIRY_TESTERS_GROUPS",
                                        description: "Array of tester groups to be notified",
-                                       default_value: []) # the default value is an empty list
+                                       default_value: []), # the default value is an empty list
+          FastlaneCore::ConfigItem.new(key: :symbols_file,
+                                       env_name: "FL_TESTFAIRY_SYMBOLS_FILE",
+                                       description: "Symbols mapping file",
+                                       default_value: Actions.lane_context[SharedValues::DSYM_OUTPUT_PATH],
+                                       verify_block: proc do |value|
+                                         UI.user_error!("Couldn't find dSYM file at path '#{value}'") unless File.exist?(value)
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :notify,
+                                       env_name: "FL_TESTFAIRY_NOTIFY",
+                                       description: "Send email to testers",
+                                       default_value: 'on'),
```

Comment:
You rather want to have them as real booleans, but then we diverge a bit from their API docs. OK from my side though.